### DISCUSSION TITLE
Reveal theme as custom format?

### DISCUSSION THREAD

**Discussion by stenw** [quarto-dev/quarto-cli#11008] [Q&A] [answered]
### Description

What exactly is the difference between creating a quarto revealjs presentation theme theme as a 'custom format'. In the documentation I think some extensions (r-ladies-theme?) are listed under both categories,

**Comment by mcanouil** [comment]
Sorry, what is the difference between what and what?

> a quarto revealjs presentation theme theme as a 'custom format'

That's one thing, what's the other?

**Reply by stenw** [reply]
The other is 'revealjs-plugin's. (so revealjs-plugin and custom format). Or maybe the two forms can converge to eachother when customizing the files.

**Comment by cderv** [comment]
> In the documentation I think some extensions (r-ladies-theme?) are listed under both categories,

I don't remember exactly why the r-ladies extension is listed under both categorie 🤔 The revealjs category is meant for [Revealjs extensions](https://quarto.org/docs/extensions/listing-revealjs.html)

Though I can say this is a custom format for revealjs that will load some default configuration in addition to the theme file. Extension can be looked at https://github.com/beatrizmilz/quarto-rladies-theme/blob/main/_extensions/rladies/_extension.yml

**Reply by stenw** [reply]
It is not only this template. For example revealjs-clean does the same.

**Reply by mcanouil** [reply, marked as answer]
The listing classification is a bit arbitrary and does not really convey a strong meaning about what is or is not a format, plugin, etc.
A custom format extension can include one or several plugins.
An extension can include multiple custom formats and many other things.
I personally removed any arbitrary classifications in my version of an extension listing: https://m.canouil.dev/quarto-extensions/
